### DISCUSSION TITLE
Can waydroid support running in a Ubuntu VM and can it be launched without GUI?

### DISCUSSION THREAD

**Discussion by cylonlxl** [waydroid/waydroid#558] [General]
Can waydroid support running in a Ubuntu VM and can it be launched without GUI?

**Comment by Runa-Chin** [comment]
If you ask that, make sure you had know [this article](https://docs.waydro.id/faq/get-waydroid-to-work-through-a-vm) firsts and if you want to want Waydroid to run Android app with window, [follow this section](https://docs.waydro.id/usage/install-on-desktops#launch-waydroid-in-full-screen-mode).

Sorry for replying your question on 2024, you have asked from 2022.
